libnetdata/log/log.h

```diff
@@ -105,6 +105,19 @@ typedef struct error_with_limit {
     usec_t sleep_ut;
 } ERROR_LIMIT;
 
+typedef enum netdata_log_level {
+    NETDATA_LOG_LEVEL_INFO,
+    NETDATA_LOG_LEVEL_ERROR,
+    NETDATA_LOG_LEVEL_ALL
+} netdata_log_level_t;
```

Comment:
I think you need from smallest to largest order. Then you can check 

- error

```c
if (NETDATA_LOG_LEVEL_ERROR > use_log_level) {
    return;
}
```

- info

```c
if (NETDATA_LOG_LEVEL_INFO > use_log_level) {
    return;
}
```

I think you get the idea

Reply:
Ahh ok :) . I was in doubt when I read the comments inside the ticket, so I started working and I would talk with you today, thanks to clarify.